site/installing.md

```diff
@@ -63,6 +65,29 @@ This script downloads and runs a recent Scope docker image from the Docker Hub.
 
 After Scope has been launched, open your web browser to [https://cloud.weave.works](https://cloud.weave.works) and login. Click 'View Instance' in the top right-hand corner to see the Scope user interface.
 
+###<a name="net-scope"></a> Local-Cluster without WeaveNet / WeaveDNS
+These are the directions for "Any Random Cluster," with no dependency on Weave Net.  Suppose I have a cluster like this:
+
+192.168.100.16
+192.168.100.17
+192.168.100.18
+192.168.100.19
+192.168.100.20
+
+We'll assume that no special hostnames or DNS settings have been fiddled with, so we're going to use IP addresses only to configure scope on this cluster:
+
+**On Each Node, first run:**
+    sudo curl -L git.io/scope -o /usr/local/bin/scope
+    sudo chmod a+x /usr/local/bin/scope
+**Then Run on the first node**
+    scope launch 192.168.100.18 192.168.100.19 192.168.100.20
+**Second node**
+    scope launch 192.168.100.17 192.168.100.20 192.168.100.21
+    scope launch 192.168.100.17 192.168.100.18 192.168.100.21
+    scope launch 192.168.100.17 192.198.100.19 192.168.100.20
+
+Key point here being that you need to make each node aware of the others.
+
 ###<a name="net-scope"></a> Weave Net and Scope
```

Comment:
(When writing docs, it's best to use second person and avoid first person. ) 

A few edits here: 

###Installing Scope onto  a Local Cluster Without Weave Net

This example assumes that you have a local cluster that is not using Weave Net, and which also has no special hostnames or DNS settings. Only the IP addresses assigned to it will be used to configure Scope.

Suppose you have the following cluster:

192.168.100.16
 192.168.100.17
 192.168.100.18
 192.168.100.19
 192.168.100.20
 
**1. First, on each node, you must run:**

     sudo curl -L git.io/scope -o /usr/local/bin/scope
     sudo chmod a+x /usr/local/bin/scope

 **2. Then on the first node:**
     scope launch 192.168.100.18 192.168.100.19 192.168.100.20

 **3. And on the second node:**

     scope launch 192.168.100.17 192.168.100.20 192.168.100.21
     scope launch 192.168.100.17 192.168.100.18 192.168.100.21
     scope launch 192.168.100.17 192.198.100.19 192.168.100.20
 
 The key point is that you will need to peer each node manually during scope launch.